resources/services/fsx/filesystems.go

```diff
@@ -0,0 +1,563 @@
+					},
+					{
+						Name:        "weekly_maintenance_start_time",
+						Description: "The preferred start time to perform weekly maintenance, formatted d:HH:MM in the UTC time zone",
+						Type:        schema.TypeString,
+					},
+				},
+			},
+			{
+				Name:        "aws_fsx_filesystem_ontap_configuration",
+				Description: "Configuration for the FSx for NetApp ONTAP file system.",
+				Resolver:    schema.PathTableResolver("OntapConfiguration"),
+				Columns: []schema.Column{
+					{
+						Name:        "filesystem_cq_id",
+						Description: "Unique CloudQuery ID of aws_fsx_filesystems table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "automatic_backup_retention_days",
+						Description: "The number of days to retain automatic backups",
+						Type:        schema.TypeBigInt,
+					},
+					{
+						Name:        "daily_automatic_backup_start_time",
+						Description: "A recurring daily time, in the format HH:MM",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "deployment_type",
+						Description: "Specifies the FSx for ONTAP file system deployment type in use in the file system.  * MULTI_AZ_1 - (Default) A high availability file system configured for Multi-AZ redundancy to tolerate temporary Availability Zone (AZ) unavailability.  * SINGLE_AZ_1 - A file system configured for Single-AZ redundancy.  For information about the use cases for Multi-AZ and Single-AZ deployments, refer to Choosing Multi-AZ or Single-AZ file system deployment (https://docs.aws.amazon.com/fsx/latest/ONTAPGuide/high-availability-multiAZ.html).",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "disk_iops_configuration_iops",
+						Description: "The total number of SSD IOPS provisioned for the file system.",
+						Type:        schema.TypeBigInt,
+						Resolver:    schema.PathResolver("DiskIopsConfiguration.Iops"),
+					},
+					{
+						Name:        "disk_iops_configuration_mode",
+						Description: "Specifies whether the number of IOPS for the file system is using the system default (AUTOMATIC) or was provisioned by the customer (USER_PROVISIONED).",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("DiskIopsConfiguration.Mode"),
+					},
+					{
+						Name:        "endpoint_ip_address_range",
+						Description: "(Multi-AZ only) The IP address range in which the endpoints to access your file system are created",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "endpoints_intercluster_dns_name",
+						Description: "The Domain Name Service (DNS) name for the file system",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("Endpoints.Intercluster.DNSName"),
+					},
+					{
+						Name:        "endpoints_intercluster_ip_addresses",
+						Description: "IP addresses of the file system endpoint.",
+						Type:        schema.TypeStringArray,
```

Comment:
IP Array?

Reply:
yes, but can't convert it now due to cq-gen quirk/ bug

Reply:
I don't think we should be prioritizing getting the data in the ideal format, rather than optimizing for what cq-gen will allow for. Users shouldn't be impacted by our automation tooling